sections/git.zsh

```diff
@@ -10,31 +13,32 @@ SPACESHIP_GIT_SHOW="${SPACESHIP_GIT_SHOW=true}"
 SPACESHIP_GIT_PREFIX="${SPACESHIP_GIT_PREFIX="on "}"
 SPACESHIP_GIT_SUFFIX="${SPACESHIP_GIT_SUFFIX="$SPACESHIP_PROMPT_DEFAULT_SUFFIX"}"
 SPACESHIP_GIT_SYMBOL="${SPACESHIP_GIT_SYMBOL=" "}"
-
-# ------------------------------------------------------------------------------
-# Dependencies
-# ------------------------------------------------------------------------------
-
-source "$SPACESHIP_ROOT/sections/git_branch.zsh"
-source "$SPACESHIP_ROOT/sections/git_status.zsh"
+if [ -z "$SPACESHIP_GIT_ORDER" ]; then
+  SPACESHIP_GIT_ORDER=(
+    branch
+    status
+  )
+fi
 
 # ------------------------------------------------------------------------------
 # Section
 # ------------------------------------------------------------------------------
 
-# Show both git branch and git status:
-#   spaceship_git_branch
-#   spaceship_git_status
 spaceship_git() {
-  [[ $SPACESHIP_GIT_SHOW == false ]] && return
+  [[ $SPACESHIP_GIT_SHOW == false || -z $SPACESHIP_GIT_ORDER ]] && return
+
+  spaceship::is_git || return
+
+  local git_prompt
 
-  local git_branch="$(spaceship_git_branch)" git_status="$(spaceship_git_status)"
+  for el in ${(Oa)SPACESHIP_GIT_ORDER}; do
```

Comment:
If I'm not mistaken this gives [revers array order](http://zshwiki.org/home/scripting/paramflags#oo_flags), right ? Would be great with a comment here.

Reply:
That's right. But now that I look at it again, I'll remove the `(Oa)` and flip line 36 to `$git_prompt$(spaceship_git_${el})`

Reply:
Addressed in c973316